flow/templates/base_script.sh

```diff
@@ -1,51 +1,51 @@
 {# The following variables are available to all scripts. #}
 {% if parallel %}
-{% set np_global = operations|map(attribute='directives.np')|sum %}
+    {% set np_global = operations|map(attribute='directives.np')|sum %}
 {% else %}
-{% set np_global = operations|map(attribute='directives.np')|max %}
+    {% set np_global = operations|map(attribute='directives.np')|max %}
 {% endif %}
 {% block header %}
-{% block preamble %}
-{% endblock %}
-{% endblock %}
+    {% block preamble %}
+    {% endblock preamble %}
+{% endblock header %}
 
 {% block project_header %}
 set -e
 set -u
 
 cd {{ project.config.project_dir }}
-{% endblock %}
+{% endblock project_header %}
 {% block body %}
-{% set cmd_suffix = cmd_suffix|default('') ~ (' &' if parallel else '') %}
-{% for operation in operations %}
+    {% set cmd_suffix = cmd_suffix|default('') ~ (' &' if parallel else '') %}
+    {% for operation in operations %}
 
 # {{ "%s"|format(operation) }}
-{% block pre_operation scoped %}
-{% endblock %}
+        {% block pre_operation scoped %}
+        {% endblock pre_operation %}
 {{ operation.cmd }}{{ cmd_suffix }}
-{% if operation.eligible_operations|length > 0 %}
+        {% if operation.eligible_operations|length > 0 %}
 # Eligible to run:
-{% for run_op in operation.eligible_operations %}
-{# The split/join handles multi-line cmd operations. #}
+            {% for run_op in operation.eligible_operations %}
+                {#- The split/join handles multi-line cmd operations. #}
```

Comment:
Curious how this syntax change relates to the new indentation?

Reply:
The `{#-` is a jinja comment that will not be output. The dash means that it will remove whitespace to its left.